modules/ai-initial-oci-resources-services.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ai-initial-oci-resources-services_{context}"]
+= Initial OCI resources and services
+
+You must create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization's requirements.
+
+Before you create {oci} resources and services, you must meet certain prerequisites that require you to configure an {oci} account that manages {oci} resources and services, and hosts the {product-title} cluster.
+
+After you meet these prerequisites, you must download an archive file from an Oracle resource. The archive file includes files for creating cluster resources and custom manifests, and a script that when run creates {oci} resources, such as DNS records, an instance, and so on.
```

Comment:
This looks like a procedure module converted into concept which basically has steps. Are we doing all of these before accessing OCI docs? There's a usage of `...must meet certain prerequisites...`. This could be ambiguous since we aren't specifying what are those. If they are in OCI docs, we could condense this information a bit more or presented more conceptually.